build/builtInExtensions.json

```diff
@@ -1,2 +1,7 @@
 [
+	{
+		"name": "Microsoft.sqlservernotebook",
+		"version": "0.3.1",
```

Comment:
And this is fixed now? Both the path length issue and the missing TSG 050 that we were seeing yesterday?

Reply:
No, I have to refresh them one more time with the latest of what Stuart has. I spoke to him offline and he said he has a PR out and once that's in, I can refresh.